pages/apim/3.x/event-native/apim-event-native-api-management-overview.adoc

```diff
@@ -0,0 +1,89 @@
+
+With this workflow, a client application running on HTTP can post content to the Gravitee Gateway. The Gateway can then take that content and push it over to Kafka as a message that Kafka can ingest. The Gravitee Gateway acts as a protocol mediation layer in this context. This enables API owners to quickly set up a sync-to-async integration.
+
+NOTE: The backend that you can post your content to via an HTTP POST entrypoint depends on the backend connectors that you have enabled via the link:{{ '/apim/3.x/apim_publisherguide_introducing_sme.html#what_is_sme' | relative_url }}[Service Management Ecosystem (SME)]. The BETA release uses the link:{{ '/apim/3.x/apim_publisherguide_introducing_kafka.html' | relative_url }}[Gravitee Community Edition / OSS Kafka connector]. In future releases we are planning to provide an advanced Kafka connector (as an Enterprise Edition feature) as well as support for more backend connectors such as MQTT, Kinesis, and others.
+
+The diagram below shows the workflow for this use case:
+
+image:{% link /images/apim/3.x/event-native/event-native-api-management-use-case-http-post.png %}[]
+
+=== Websocket and SSE entrypoints
+
+Support for Websocket (Community Edition/OSS) and Server-Sent Events (Enterprise Edition only) entrypoints are provided as separate plugins.
+
+NOTE: These plugins are expected shortly after the main 3.19 release and will be announced here when available.
+
+This feature enables you to front your event-driven backend - currently only using Kafka via the link:{{ '/apim/3.x/apim_publisherguide_introducing_kafka.html' | relative_url }}[Gravitee Community Edition / OSS Kafka connector] - with Websocket or Server-Sent Events so that API consumers can access events from the Kafka backend via one of these more consumer-friendly protocols.
+
+If an API consumer cannot connect to a native Kafka instance due to technical limitations (for example, if they cannot implement a Kafka client themselves), you can make Kafka topics available to that consumer via another, more consumer-friendly protocol such as SSE or Websocket.
+
+The diagram below shows the workflow for this use case:
+
+image:{% link /images/apim/3.x/event-native/event-native-api-management-use-case-event-consumption-streaming.png %}[]
+
+This feature enables you to potentially cover use cases in the fields of Event-Driven Architecture (EDA) and Internet of Things (IoT) without the Gateway and API Management components becoming a bottleneck.
+
+NOTE: In comparison to the link:{{ '/apim/3.x/apim_event_native_api_management_overview.html#http_post_entrypoint' | relative_url }}[HTTP POST entrypoint use case] using Kafka (where the HTTP POST method is just about sending information from the client app to Kafka), with SSE and Websocket it is possible for the client app to receive information from Kafka as well.
```

Comment:
In case of websocket, actually you can do both: Publish and Subscribe at the same time.

Reply:
@guillaumelamirand isn't this already the meaning in the note?

Reply:
The note is saying that with SSE and websocket you could receive information from kafka, but doesn't mention the fact that websocket is actually bidirectional.